geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl

```diff
@@ -18,86 +18,99 @@ def call(verbose, arguments, **kwargs):
+    """
+    env = {}
+    for env_file in env_files:
+        with open(env_file) as f:
+            for line in f:
+                if line and line[0] != "#":
+                    try:
+                        index = line.index("=")
+                        env[line[:index].strip()] = line[index + 1 :].strip()
+                    except ValueError:
+                        # Ignore lines that don't have a '='
+                        pass
+    return env
 
-args = parser.parse_args()
 
-if args.upgrade:
-    version_re = re.search(r"([0-9]\.[0-9])\.[0-9]", args.upgrade)
-    major_version = version_re.group(0) if version_re is not None else args.upgrade
-    full_version = args.upgrade if args.upgrade != "master" else "{{geomapfish_main_version}}"
-    with open("upgrade", "w") as f:
-        result = urllib.request.urlopen(
-            "https://raw.githubusercontent.com/camptocamp/c2cgeoportal/{}/scripts/upgrade".format(
-                major_version
+def main():
+    parser = argparse.ArgumentParser(description="Build the project")
+    parser.add_argument("--verbose", help="Display the docker build commands")
+    parser.add_argument("--config", action="store_true", help="Build only the configuration image")
+    parser.add_argument("--geoportal", action="store_true", help="Build only the geoportal image")
+    parser.add_argument("--upgrade", help="Start upgrading the project to version")
+    parser.add_argument("env_files", nargs="*", help="The environment files")
+    args = parser.parse_args()
+
+    if args.upgrade:
+        version_re = re.search(r"([0-9]\.[0-9])\.[0-9]", args.upgrade)
+        major_version = version_re.group(0) if version_re is not None else args.upgrade
+        full_version = args.upgrade if args.upgrade != "master" else "{{geomapfish_main_version}}"
+        with open("upgrade", "w") as f:
+            result = urllib.request.urlopen(
+                "https://raw.githubusercontent.com/camptocamp/c2cgeoportal/{}/scripts/upgrade".format(
+                    major_version
+                )
             )
-        )
-        if result.code != 200:
-            print("ERROR:")
-            print(result.read())
+            if result.code != 200:
+                print("ERROR:")
+                print(result.read())
+                sys.exit(1)
+            f.write(result.read().decode())
+        os.chmod("upgrade", os.stat("upgrade").st_mode | stat.S_IXUSR)
+        try:
+            subprocess.check_call(["./upgrade", full_version])
+        except subprocess.CalledProcessError:
             sys.exit(1)
-        f.write(result.read().decode())
-    os.chmod("upgrade", os.stat("upgrade").st_mode | stat.S_IXUSR)
-    try:
-        subprocess.check_call(["./upgrade", full_version])
-    except subprocess.CalledProcessError:
-        sys.exit(1)
-    sys.exit(0)
+        sys.exit(0)
+
+    env_files = args.env_files or load_env(['.env'])['ENV_FILES'].split(" ")
```

Comment:
Why the or ...?

Reply:
This is a default value
if args.env_files != [] => args.env_files
else load_env(['.env'])['ENV_FILES'].split(" ")

Reply:
if user do not give any env_files, use previous ones.